ruby/extractor/src/extractor.rs

```diff
@@ -383,15 +401,13 @@ impl<'a> Visitor<'a> {
                 }
             }
             _ => {
-                let error_message = format!("unknown table type: '{}'", node.kind());
                 self.record_parse_error(
                     loc,
                     self.diagnostics_writer
-                        .message("parse-error", "Parse error")
+                        .new_entry("parse-error", "Parse error")
                         .severity(diagnostics::Severity::Error)
                         .location(self.path, start_line, start_column, end_line, end_column)
-                        .text(&error_message)
-                        .status_page(),
+                        .message("unknown table type: {}", &[node.kind()]),
```

Comment:
I notice an inconsistency in the capitalization of the message text - is there a convention or style guide to refer to about this?